instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -167,6 +175,18 @@ def _instrumented_requests_call(
             SpanAttributes.HTTP_URL: url,
         }
 
+        metric_labels = {
+            "http.method": method,
+            "http.url": url,
+        }
+
+        try:
+            parsed_url = urlparse(url)
```

Comment:
I imagine that the call to `urlparse` is what can cause the `ValueError` exception to be raised. If that happens, does it make sense to continue the instrumentation with a `metrics_labels` dictionary that will not have `"http.host"` and `"http.scheme"` keys?

Reply:
In traces we add the url even if the url is invalid (e.g. `random123` is also added as span attribute). So, I added it here for consistency.
I think its okay to go without `http.host` and `http.scheme` keys as `http.url` attribute alone is a recommended set of attributes for client metrics (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attribute-alternatives)